### DISCUSSION TITLE
Is it possible to disable a delimiter for an allowed filter?

### DISCUSSION THREAD

**Discussion by ljjackson** [spatie/laravel-query-builder#756] [Q&A]
This package, as far as I can tell does not allow for the developer to disable a delimiter for a particular allowed filter, I believe this should be in the scope of the package.

For example in our system we have a search term filter, this allows for any input from the user therefore applying an obscure delimiter could potentially end up with the user receiving a 500 response.

The workaround we're doing at the moment is joining the search term back together based on the delimiter.

Before I submit a PR for this, is it possible to disable the delimiter already and the docs just need some TLC, or is this outside of the scope of the package?

Thanks in advance!

**Comment by stetim94** [comment]
I know this ticket is very old, but the answer is yes:
```
                AllowedFilter::custom('abc', new Filter(['field']), null, false),
```
by giving `false` as value/argument for `arrayValueDelimiter`, you disable the delimiter.
